tests/providers/odbc/hooks/test_odbc.py

```diff
@@ -235,3 +259,15 @@ def test_sqlalchemy_scheme_extra(self):
         hook = self.get_hook(conn_params=dict(extra=json.dumps(dict(sqlalchemy_scheme="my-scheme"))))
         uri = hook.get_uri()
         assert urlsplit(uri).scheme == "my-scheme"
+
+    def test_query_return_value_serialization(self, Row):
```

Comment:
Maybe we can also add a testcase for `return_serializable=False`?

Reply:
Added it